react/i18n/l10n/sv.json

```diff
@@ -58,6 +59,8 @@
   "error_lookup_mobile_task": "Problem looking up the phone number",
   "error_navet_task": "Communication problem with Navet",
   "faq_link": "För mer information, se ",
+  "foot.change-version": "Use old version of the app",
+  "foot.change-version-tip": "This is an experimental version. If you meet any problem while using the app, you can switch to the old version.",
```

Comment:
meet -> experience

Reply:
In this and the next I disagree - it should be swedish ;-)

But of course I'll correct them for the benefit of the translator :)